lib/actions/reassemble.js

```diff
@@ -43,6 +54,11 @@ async function processAction(msg, cfg) {
     messageData,
   };
 
+  // chcek if messageId exist
+  if (!messageId) {
+    object.messageId = randonGUID();
+  }
+
```

Comment:
Can you change line 44-49 to 
```
  const {
    groupSize,
    groupId,
    messageId = randonGUID(),
    messageData,
  } = msg.body;
```
instead?

Reply:
done!